frontend/src/components/scratchpad/scratch-pad.tsx

```diff
@@ -0,0 +1,91 @@
+/* Copyright 2024 Marimo. All rights reserved. */
+import type React from "react";
+import {
+  LastSavedNotebook,
+  flattenNotebookCells,
+  useCellActions,
+  useNotebook,
+} from "@/core/cells/cells";
+import {
+  getAppConfig,
+  getUserConfig,
+  useUserConfig,
+} from "@/core/config/config";
+import { useState } from "react";
+import { useAtom } from "jotai";
+import { viewStateAtom } from "@/core/mode";
+import { useTheme } from "@/theme/useTheme";
+import { Cell } from "../editor/Cell";
+import { CellId } from "@/core/cells/ids";
+export const ScratchPad: React.FC = () => {
+  const notebook = useNotebook();
+  const [lastSavedNotebook, setLastSavedNotebook] =
+    useState<LastSavedNotebook>();
+  const [viewState, setViewState] = useAtom(viewStateAtom);
+  const { theme } = useTheme();
+
+  const {
+    updateCellCode,
+    prepareForRun,
+    moveCell,
+    moveToNextCell,
+    updateCellConfig,
+    clearSerializedEditorState,
+    focusCell,
+    createNewCell,
+    focusBottomCell,
+    focusTopCell,
+    scrollToTarget,
+    foldAll,
+    unfoldAll,
+    sendToBottom,
+    sendToTop,
+    setStdinResponse,
+  } = useCellActions();
+  const cells = flattenNotebookCells(notebook);
+  const testCell = cells[0];
+  return (
+    <div className="p-2">
+      <Cell
```

Comment:
I think ideally we can create a single scratch cell that doesn't exist in the notebook graph but can still be executed. I think we can hard-code an ID (i.e. `__scratch__` so that the backend knows to run this cell, but not include its definitions as part of the dependency graph. 

(we can own some of that backend logic if you prefer)

Reply:
I was kinda hoping someone would drop in here and tell me what I was doing wrong 😆 

Gonna take a look at OutputArea rn

Reply:
How would you recommend creating this cell? It seems `createNewCell` wouldn't really be appropriate because it has logic like assigining ID and adding to the array which we want to avoid. Should I add a new action for creating scratchpad cells?